htmd/data/test-param/h2o2_esp/energies.txt

```diff
@@ -5,7 +5,7 @@ Bond     : 0.09165582558617029
 Angle    : 0.13882333179780315
 Dihedral : 3.1999999910373385
 Improper : 0.0
-Electro  : 20.593075751152295
+Electro  : 5.7857768770164375
```

Comment:
What was going wrong before with ESP that now the fixed version provides such a different electrostatic energy?

Reply:
Hum, I see, the partial charges difference between oxygens and hydrogens went down.

Reply:
This is due to the unit conversion bug!